### PR TITLE
feat(profile): add dir flag

### PR DESCRIPTION
**Pull request by Naisisor**

### CODE REVIEW

File: cmd/profile.go [hunk 1]
```diff
@@ -66,6 +66,15 @@ goc profile --force
 		if output == "" {
 			fmt.Fprint(os.Stdout, string(res))
 		} else {
+
+			if dir != "" {
```
Comment (CarlJi) [inline cmd/profile.go:70]: Good requirements, but no need to add new flag. It's better to support like these:
`goc profile -o /path/to/file`
`goc profile -o relative-path/to/file`

Comment (Naisisor) [reply cmd/profile.go:70]: I think

-o: Write the log information to the specified file
-d: Save the file to the specified path or create it if it does not exist

They have different functions

Comment (CarlJi) [reply cmd/profile.go:70]: i prefer the behavior following command `go build [-o output]`, refer: https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies

Comment (CarlJi) [reply cmd/profile.go:70]: i prefer the behavior following command `go build [-o output]`, refer: https://golang.org/cmd/go/#hdr-Compile_packages_and_dependencies

Comment (Naisisor) [reply cmd/profile.go:70]: Thank you. I understand

**Pull request closed without merging**